maven_plugin/src/test/resources-its/it/IntegrationTestsIT/singleDependencyCheckMustFail/lockfile.json

```diff
@@ -79,18 +84,19 @@
   "mavenPlugins": [],
   "metaData": {
     "environment": {
-      "osName": "Mac OS X",
-      "mavenVersion": "3.8.2",
+      "osName": "Linux",
+      "mavenVersion": "3.9.9",
       "javaVersion": "21.0.5"
     },
     "config": {
       "includeMavenPlugins": false,
       "allowValidationFailure": false,
+      "allowPomValidationFailure": true,
```

Comment:
based on the docs above, it seems this should  only warn when `allowPomValidationFailure` is set to `true`.